proj/shapeless-java-records.conf

```diff
@@ -2,7 +2,7 @@
 
 vars.proj.shapeless-java-records: ${vars.base} {
   name: "shapeless-java-records"
-  uri: "https://github.com/xuwei-k/shapeless-java-records.git#4a90789e686e55218616a83a1f2b8df8b0151319"
+  uri: "https://github.com/xuwei-k/shapeless-java-records.git#a7c5da9b6ef6afa20dac4fc55865e4a20016dfb7"
```

Comment:
note:
This version require JDK15

- https://github.com/xuwei-k/shapeless-java-records/commit/dd94a7c0bf39201bf9772e3602feb90c23f927d7
- https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/6058/consoleText

```
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:43:15: value isSealed is not a member of Class[?0]
[shapeless-java-records] [error]     if (clazz.isSealed || clazz.isRecord) {
[shapeless-java-records] [error]               ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:43:33: value isRecord is not a member of Class[?0]
[shapeless-java-records] [error]     if (clazz.isSealed || clazz.isRecord) {
[shapeless-java-records] [error]                                 ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:45:19: value isRecord is not a member of Class[?0]
[shapeless-java-records] [error]         if (clazz.isRecord) {
[shapeless-java-records] [error]                   ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:46:17: value getRecordComponents is not a member of Class[?0]
[shapeless-java-records] [error]           clazz.getRecordComponents.map(_.getName)
[shapeless-java-records] [error]                 ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:43:15: value isSealed is not a member of Class[?0]
[shapeless-java-records] [error]     if (clazz.isSealed || clazz.isRecord) {
[shapeless-java-records] [error]               ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:48:17: value permittedSubclasses is not a member of Class[?0]
[shapeless-java-records] [error]           clazz.permittedSubclasses.map(_.displayName)
[shapeless-java-records] [error]                 ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:43:33: value isRecord is not a member of Class[?0]
[shapeless-java-records] [error]     if (clazz.isSealed || clazz.isRecord) {
[shapeless-java-records] [error]                                 ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:45:19: value isRecord is not a member of Class[?0]
[shapeless-java-records] [error]         if (clazz.isRecord) {
[shapeless-java-records] [error]                   ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:46:17: value getRecordComponents is not a member of Class[?0]
[shapeless-java-records] [error]           clazz.getRecordComponents.map(_.getName)
[shapeless-java-records] [error]                 ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:48:17: value permittedSubclasses is not a member of Class[?0]
[shapeless-java-records] [error]           clazz.permittedSubclasses.map(_.displayName)
[shapeless-java-records] [error]                 ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:87:26: value permittedSubclasses is not a member of Class[?0]
[shapeless-java-records] [error]     val subTypes = clazz.permittedSubclasses.map { subClassDesc =>
[shapeless-java-records] [error]                          ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:87:26: value permittedSubclasses is not a member of Class[?0]
[shapeless-java-records] [error]     val subTypes = clazz.permittedSubclasses.map { subClassDesc =>
[shapeless-java-records] [error]                          ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:122:15: value isSealed is not a member of Class[?0]
[shapeless-java-records] [error]     if (clazz.isSealed) {
[shapeless-java-records] [error]               ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:124:22: value isRecord is not a member of Class[?0]
[shapeless-java-records] [error]     } else if (clazz.isRecord) {
[shapeless-java-records] [error]                      ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:122:15: value isSealed is not a member of Class[?0]
[shapeless-java-records] [error]     if (clazz.isSealed) {
[shapeless-java-records] [error]               ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:124:22: value isRecord is not a member of Class[?0]
[shapeless-java-records] [error]     } else if (clazz.isRecord) {
[shapeless-java-records] [error]                      ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:144:34: value getRecordComponents is not a member of Class[?0]
[shapeless-java-records] [error]     val recordComponents = clazz.getRecordComponents.toList
[shapeless-java-records] [error]                                  ^
[shapeless-java-records] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/shapeless-java-records-a4b8b35507fbb72b298b736e3ede335288943632/src/main/scala/JavaRecordGeneric.scala:144:34: value getRecordComponents is not a member of Class[?0]
[shapeless-java-records] [error]     val recordComponents = clazz.getRecordComponents.toList
[shapeless-java-records] [error]                                  ^
[shapeless-java-records] [error] 9 errors found
[shapeless-java-records] [info] No documentation generated with unsuccessful compiler run
[shapeless-java-records] [error] 9 errors found
[shapeless-java-records] [error] (Compile / compileIncremental) Compilation failed
[shapeless-java-records] [error] (Compile / doc) Scaladoc generation failed
```